keras/applications/mobilenet.py

```diff
@@ -495,78 +607,3 @@ def _depthwise_conv_block(input, pointwise_conv_filters, alpha,
     x = Activation(relu6, name='conv_pw_%d_relu' % id)(x)
 
     return x
-
-
-def _create_mobilenet(classes, img_input, include_top, alpha, depth_multiplier, dropout, pooling):
```

Comment:
Ah, I responded to that. To re-iterate : 
I generally agree to what you are saying. There are some very finicky models (like DenseNets and DenseNetFCNs) which really require their own separate methods, simply because of the large variety of very similar models that can be built due to complex hyper-parameters which interact and conflict with one another.

This however is a special case, where the model is completely linear and doesn't even have residual connections. It doesn't really hurt much to inline it.